Don't require twine. Twine is for uploading to pypi which we're not doing.
https://github.com/dtrx-py/dtrx/issues/19
--- setup.py.orig	2021-09-15 15:45:01.000000000 -0500
+++ setup.py	2022-01-07 16:28:35.000000000 -0600
@@ -70,6 +70,6 @@
     long_description_content_type="text/markdown",
     # using markdown as pypi description:
     # https://dustingram.com/articles/2018/03/16/markdown-descriptions-on-pypi
-    setup_requires=["setuptools>=38.6.0", "wheel>=0.31.0", "twine>=1.11.0"],
+    setup_requires=["setuptools>=38.6.0", "wheel>=0.31.0"],
     install_requires=install_requires,
